community-containers/npmplus/npmplus.json

```diff
@@ -0,0 +1,30 @@
+{
+    "aio_services_v1": [
+        {
+            "container_name": "nextcloud-aio-npmplus",
+            "display_name": "NPMplus",
+            "documentation": "https://github.com/nextcloud/all-in-one/tree/main/community-containers/npmplus",
+            "image": "zoeyvid/npmplus",
+            "image_tag": "latest",
+            "internal_port": "host",
+            "restart": "unless-stopped",
+            "environment": [
+                "TZ=%TIMEZONE%",
+                "NC_AIO=true",
+                "NC_DOMAIN=%NC_DOMAIN%",
+                "NPM_LISTEN_LOCALHOST=true"
+            ],
+            "volumes": [
+                {
+                    "source": "nextcloud_aio_npmplus",
```

Comment:
Does this have some data that should be backed up? 

If so it can be covered like this https://github.com/nextcloud/all-in-one/blob/5dcdb6268ab2b08b58f7f2510de3bfe01bc55bbc/community-containers/plex/plex.json#L36-L38

It will also get restored on a new instance then automatically :)

Reply:
yes should be inside the backup

Reply:
all right :)